pkg/webhooks/resource/handlers.go

```diff
@@ -181,13 +181,13 @@ func (h *handlers) Mutate(logger logr.Logger, request *admissionv1.AdmissionRequ
 		return admissionutils.ResponseFailure(err.Error())
 	}
 	newRequest := patchRequest(mutatePatches, request, logger)
+	ivh := imageverification.NewImageVerificationHandler(logger, h.kyvernoClient, h.eventGen, h.admissionReports)
```

Comment:
Why move it up here?

From this message it looks like we need to rebuild the policy context for imageVerify rules:
```
// rebuild context to process images updated via mutate policies
```

Reply:
mistake ? i don't remember i moved it...

Reply:
Moved back to the original line